compiler/rustc_parse/src/parser/path.rs

```diff
@@ -598,5 +637,6 @@ impl<'a> Parser<'a> {
                 return Err(err);
             }
         }
+            */
```

Comment:
Please clean up commented out code and if possible, add comments as to why we're doing the extra logic :)